templates/components/table/row/actions.html.twig

```diff
@@ -1,18 +1,15 @@
-<td class="flex items-center justify-center">
-  {% for key, action in config.actions %}
-    {% if action.validation is defined %}
-      <label 
-        data-action="click->table#action"
-        for="action-modal" class="btn btn-{{ key }} ml-1" 
-        data-route="{{ path(action.route, { id: config.value }) }}" 
-        data-message="{{ action.validation.message|trans({}, 'table+intl-icu') }}"
-      >
-        {% include 'icon/' ~ action.icon ~ '.html.twig' %}
-      </label>
-    {% else %}
-      <a class="btn btn-{{ key }} ml-1" href="{{ path(action.route, { id: config.value }) }}">
-        {% include 'icon/' ~ action.icon ~ '.html.twig' %}
-      </a>
-    {% endif %}
-  {% endfor %}
-</td>
\ No newline at end of file
+<td class="flex gap-2">
+    {% for key, action in config.actions %}
+        {% if action.validation is defined %}
+            <label data-action="click->table#action"
+                for="action-modal"
```

Comment:
Le `for` vise toujours la même modal c'est normal?

Reply:
Ouais c'est normal, j'ai fait en sorte que mon système utilise une seule modal pour pas en avoir un nombre trop élevé en fonction des lignes du tableau